src/app/pages/tutorial/tutorial.page.scss

```diff
@@ -9,9 +9,28 @@ ion-content {
           position: absolute;
           left: 50%;
           bottom: 15px;
-          width: 100%;
-          height: 60px;
+          width: 80%;
+          height: 50px;
           transform: translate(-50%, 0);
+          align-items: center;
+          padding: 30 30px;
+          border: 1px solid #564dfc;
+          box-sizing: border-box;
+          border-radius: 25px;
+        }
+
+        button {
```

Comment:
Two rules `button[routerLink]` and `button` are essentially the same. Merge them.